main.go

```diff
@@ -81,7 +81,7 @@ func main() {
 
 	envSource, err := prefixcollector.NewEnvPrefixSource(config.ExcludedPrefixes)
 	if err != nil {
-		span.Logger().Fatalf("Failed to parse prefixes from environment: ", err)
+		span.Logger().Fatalf("Failed to parse prefixes from environment: %v", err)
```

Comment:
Why we cannot call fatailf in the env source?

Reply:
I thought it would be strange, if source constructor will be able to terminate application.

Reply:
Then you can check prefixes before create collector

Reply:
Did you mean before creating source? If so, fixed.